charts/kof-storage/values.yaml

```
@@ -69,6 +69,10 @@ grafana:
     host: grafana.kcm0.example.net
   autoinstrumentation:
     enabled: true
+  dashboard:
+    datasource:
+      regex: ""
+    filters: []
```

```suggestion
    filters: {}
```
It should be a dictionary, not a list I assume

---

@AndrejsPon00 please apply this last suggestion and we're good to go!

---

Done 👍